src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -226,6 +226,15 @@ UserImportResult importUsers(UserImportRequest request) throws FirebaseAuthExcep
     return new UserImportResult(request.getUsersCount(), response);
   }
 
+  void deleteTenant(String tenantId) throws FirebaseAuthException {
+    GenericUrl url = new GenericUrl(tenantMgtBaseUrl + "/tenants:delete");
+    url.put("name", tenantId);
```

Comment:
Should use `DELETE /tenants/{tenantId}` URL format here.

Reply:
Done.